apps/accounts/models/automations/base.py

```diff
@@ -51,7 +51,7 @@ def start(self):
 class ChangeSecretMixin(SecretWithRandomMixin):
     ssh_key_change_strategy = models.CharField(
         choices=SSHKeyStrategy.choices, max_length=16,
-        default=SSHKeyStrategy.add, verbose_name=_('SSH key change strategy')
+        default=SSHKeyStrategy.set_jms, verbose_name=_('SSH key change strategy')
     )
     get_all_assets: callable  # get all assets
 
```

Comment:
There is no known irregularity or issue with this provided code snippet as of October 10, 2023. It appears to be functional and follows basic programming standards for Django models.

If there were discrepancies that needed discussion or changes based on the knowledge cut-off I used (September 1st) then we would need more details about what kind of differences you're referring to. This depends specifically on any recent updates to Django or other external libraries the model uses. Please clarify your request so I can provide a precise answer!